pkg/scheduler/objects/application.go

```diff
@@ -841,6 +841,21 @@ func (sa *Application) tryPlaceholderAllocate(nodeIterator func() interfaces.Nod
 					log.Logger().Warn("ask repeat update failed unexpectedly",
 						zap.Error(err))
 				}
+				// Is real allocation matches with placeholder resource?
+				if ! resources.Equals(alloc.AllocatedResource, ph.AllocatedResource) {
+					// post scheduling events via the event plugin
+					if eventCache := events.GetEventCache(); eventCache != nil {
+						message := fmt.Sprintf("Real Pod allocation %s is not matching with placeholder allocation %s in application %s", request.AllocationKey, ph.AllocationKey, request.ApplicationID)
```

Comment:
Could you please add in the pod event numerical values of the resources? Something like "real resource: %s, placeholder resource: %s"?